pkg/eventcounter/api/api.go

```diff
@@ -53,9 +53,11 @@ import (
 const listRequestPageSize = 500
 
 const (
-	eventCountPrefix   = "ec"
-	userCountPrefix    = "uc"
-	defaultVariationID = "default"
+	eventCountPrefix             = "ec"
+	userCountPrefix              = "uc"
+	opsEvaluationUserCountPrefix = "ops_euc"
+	opsGoalUserCountPrefix       = "ops_guc"
```

Comment:
I’d like to match the settings on the Persister side.
And, it might be better that we write these const strings in `pkg/cache/cache.go`.